Files/sans_bonestab1.csv

```diff
@@ -1,26 +1,105 @@
-0,CombatZoneResize,241,226,406,391,TLResume
-0,HeartTeleport,320,304,,,
-0,HeartMode,0,,,,
-0,TLPause,,,,,
-0,SET,Loop,9,,,
-0,JMPZ,26,$Loop,,,
-0,SUB,Loop,$Loop,1,,
-0,RND,Direction,4,,,
-0,ADD,Jump,$Direction,1,,
-0,JMPREL,$Jump,,,,
-0,JMPREL,4,,,,
-0,JMPREL,5,,,,
-0,JMPREL,6,,,,
-0,JMPREL,7,,,,
-0,SansBody,HandRight,,,,
-0,JMPREL,7,,,,
-0,SansBody,HandDown,,,,
-0,JMPREL,5,,,,
-0,SansBody,HandLeft,,,,
-0,JMPREL,3,,,,
-0,SansBody,HandUp,,,,
-0,JMPREL,1,,,,
-0.26666,SansSlam,$Direction,,,,
-0.2,BoneStab,$Direction,25,0.4,0.33333,
-0.6,JMPABS,6,,,,
-0,EndAttack,,,,,
+0,Cosmetics,Phase2
+0,CombatZoneResizeInstant,239,226,404,387,,,,
+0,HeartTeleport,320,304,,,,,,
+0,HeartMode,0,,,,,,,
+0.2,SansText,test,,,
+0,HeartMode,1,,,,,,,
+0.5,BoneVRepeat,128,365,20,0,300,16,170
+0,BoneVRepeat,503,365,20,2,300,16,170
+1,BoneV,239,100,500,4,250,2
+0,BoneV,390,100,500,2,250,2
+1,BoneV,239,100,500,4,250,2
+0,BoneV,390,100,500,2,250,2
+1,BoneV,239,100,500,4,250,2
+0,BoneV,390,100,500,2,250,2
+1,BoneV,239,100,500,4,250,2
+0,BoneV,390,100,500,2,250,2
+1,BoneV,239,100,500,4,250,2
+0,BoneV,390,100,500,2,250,2
+1,BoneV,239,100,500,4,250,2
+0,BoneV,390,100,500,2,250,2
+1,BoneV,239,100,500,4,250,2
+0,BoneV,390,100,500,2,250,2
+1,BoneV,239,100,500,4,250,2
+0,BoneV,390,100,500,2,250,2

```

Comment:
You can use BoneVRepeat because I changed it so you can use colors with BoneVRepeat with colors

Reply:
hello fredo it's me portalaura

i need a fight full of orange bone and blasters